src/app/Archives/Archives.tsx

```diff
@@ -63,7 +64,7 @@ export const Archives = () => {
   notification handling in the ArchivedRecordingsTable. 
   */ 
   const target: Target = { 
-    connectUrl: '',
+    connectUrl: UPLOADS_SUBDIRECTORY,
```

Comment:
I am just a bit concerned that within `ArchivedRecordingsTable`, we call to `context.target` for selected target at some hooks:

https://github.com/cryostatio/cryostat-web/blob/bb8b4db69ada370dd654ca2ddc54bdc44d8448ee/src/app/Recordings/ArchivedRecordingsTable.tsx#L247

but use `props.target` at others:

https://github.com/cryostatio/cryostat-web/blob/bb8b4db69ada370dd654ca2ddc54bdc44d8448ee/src/app/Recordings/ArchivedRecordingsTable.tsx#L228

Also, this could complicate more as now recording table saves a targetConnectURl as a local state.

Reply:
I think the `props.target` passed to ArchivedRecordingTable mostly for UploadsTable as it does not have target. I think we could either:

1. Create a separate component for UploadsTable
2. Rename props and add checks if we should use `context.target` or fixed `props.target`).

Reply:
I think this is problem is a bit big to be part of this issue, I see what you mean about wanting the UploadsTable to be another component as the layout right now is a bit messy.